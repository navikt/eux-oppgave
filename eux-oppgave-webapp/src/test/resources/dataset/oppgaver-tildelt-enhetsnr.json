{
  "versjon": 4,
  "tildeltEnhetsnr": "2950",
  "kommentar": "en kommentar"
}