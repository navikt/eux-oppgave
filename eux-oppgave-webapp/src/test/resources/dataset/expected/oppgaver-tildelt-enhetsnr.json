{
  "versjon": 4,
  "tilordnetRessurs": null,
  "tildeltEnhetsnr": "2950",
  "kommentar": {
    "tekst": "en kommentar",
    "automatiskGenerert": false
  },
  "beskrivelse": "--- 05.08.2024 12:39 (ukjent) ---\nen kommentar\n\ndesc"
}
